Tests/src/TestHelpers.h

```diff
@@ -162,6 +163,20 @@ inline void SetRandSeed()
 {
 	std::srand(static_cast<unsigned int>(std::time(nullptr)));
 }
+
+inline double RandomUniformDouble()
```

Comment:
It could, but my preference would be for simple-to-use and easy-to-understand utility random functions. I guess we could add more bespoke ones in the future?

Reply:
I decided you were right, and have introduced a test helper function `RandomRangeDouble(double Min, double Max)` to the PR.